sbnanaobj/StandardRecord/SRBNBInfo.h

```diff
@@ -0,0 +1,52 @@
+#ifndef SRBNBINFO_h
+#define SRBNBINFO_h
+
+#include <vector>
+
+namespace caf
+{
+  class SRBNBInfo
+  {
+  public:
```

Comment:
I worry this causes trouble in SRProxy. Some picky compilers won't allow you to forward-declare struct as class and vice versa, and it's easy to do that by accident.

Reply:
Is it human-driven problems (like "people are used to forward-declare with `class`), or some machine problem ("the parser/code generator expects `class`")?

Reply:
Machine. I get a list of "classes" which I think also includes structs, and then I generate code for them as if they're classes.
I've also written the human problem myself, and it's particularly annoying because gcc lets you do it but clang doesn't (I think).

Reply:
Yes, GCC 9.3 forgives, Clang 7 does not.
Then we should update [CF-159] accordingly.